flask_opsgenie/__init__.py

```diff
@@ -130,23 +133,34 @@ def _after_request(self, response: Response):
         status_code = status_code.value if type(status_code) == HTTPStatus else status_code
         status_class = self._get_status_class(status_code)
         endpoint = request.path
+        extra_props = {}
+
+        # Fetching Request header values
+        if self._forwarded_header_keys:
```

Comment:
@msg4sunny since this same code construct is being used below as well for header value extraction, can we move it to a utility function and use it in different places like ```_extract_request_headers```.

Reply:
Good idea!. Changes are done.